### PR TITLE
[W5.11][T16-A2] Le Khang Tai

### PR DESCRIPTION
**Pull request by LeKhangTai**
Second enhancement - Added Sort function for sorting persons names

### CODE REVIEW

**Comment by LeKhangTai** [comment]
Hi,

I made a careless mistake by branching this file from the first enhancement I made - Case insensitivity. Hence, you will see duplicate files being committed.  However, test cases have been updated to fulfill both enhancements.

File: src/seedu/addressbook/commands/HelpCommand.java [hunk 1]
```diff
@@ -21,6 +21,7 @@ public CommandResult execute() {
                 + "\n" + ClearCommand.MESSAGE_USAGE
                 + "\n" + FindCommand.MESSAGE_USAGE
                 + "\n" + ListCommand.MESSAGE_USAGE
+                        + "\n" + SortCommand.MESSAGE_USAGE
```
Comment (asiddharth) [inline src/seedu/addressbook/commands/HelpCommand.java:24]: Indentation problem here?

File: src/seedu/addressbook/commands/SortCommand.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+import java.util.List;
+
+
```
Comment (asiddharth) [inline src/seedu/addressbook/commands/SortCommand.java:7]: Missing header comment. All classes should have one.

File: src/seedu/addressbook/commands/SortCommand.java [hunk 3]
```diff
@@ -0,0 +1,25 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+import java.util.List;
+
```
Comment (asiddharth) [inline src/seedu/addressbook/commands/SortCommand.java:6]: Extra blank line. Use blank lines sparingly. It's good to have blank lines to separate logical code blocks, but too many blank lines will increase the height of the code, which increases the amount of scrolling.

File: src/seedu/addressbook/parser/Parser.java [hunk 4]
```diff
@@ -90,6 +91,9 @@ public Command parseCommand(String userInput) {
         case ListCommand.COMMAND_WORD:
             return new ListCommand();
 
+            case SortCommand.COMMAND_WORD:
```
Comment (asiddharth) [inline src/seedu/addressbook/parser/Parser.java:94]: Indentation problem here?

**Review by asiddharth — COMMENTED** [review]
@LeKhangTai  Some comments added. Please close the PR after reading comments. Please make sure clean PRs are submitted from the next LO. Unclean PRs will not be reviewed again.

**Pull request closed without merging**